### PR TITLE
Create CITATION.cff

### PR DESCRIPTION
**Pull request by joecummings**
👀

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,9 @@
+cff-version: 1.2.0
+title: "torchtune: the PyTorch native finetuning library"
```
Comment (SalmanMohammadi) [inline CITATION.cff:2]:
```suggestion
title: "torchtune: a PyTorch native finetuning library"
```

Comment (joecummings) [reply CITATION.cff:2]: You don't like being THE finetuning library?? 🥹

**Comment by joecummings** [comment]
> Should we also add a section in the README?

added

**Comment by SalmanMohammadi** [comment]
"a finetuning library" bestows us with a modest sagaciousness

Think we just need to link to the section in the readme heading (https://github.com/pytorch/torchtune/blob/a8a64ec6a99a6ea2be4fdaf0cd5797b03a2567cf/README.md?plain=1#L10)

**Review by SalmanMohammadi — APPROVED** [review]

File: CITATION.cff (hunk 1 above)
Comment (ebsmothers) [reply CITATION.cff:2]: Or we can split the difference and say "PyTorch's finetuning library"

**Pull request merged**